packages/fast-development-site-react/src/components/site/dev-tools-code-preview-style.ts

```diff
@@ -0,0 +1,201 @@
+export default {
+    "code[class*=\"language-\"]": {
+        "color": "#464646",
+        "fontFamily": "\"Consolas\", \"Bitstream Vera Sans Mono\", \"Courier New\", Courier, monospace",
+        "direction": "ltr",
+        "textAlign": "left",
+        "whiteSpace": "pre",
+        "wordSpacing": "normal",
+        "wordBreak": "normal",
+        "fontSize": "0.95em",
+        "lineHeight": "1.2em",
+        "MozTabSize": "4",
+        "OTabSize": "4",
+        "tabSize": "4",
+        "WebkitHyphens": "none",
+        "MozHyphens": "none",

```

Comment:
Should any of these start with an uppercase letter? Can we confirm this if it was copy/pasted?